config/repository_config.json

```diff
@@ -13,7 +13,7 @@
    "DEVELOPMENTSTATUS" : "Development Status :: 4 - Beta",
    "INTENDEDAUDIENCE" : "Intended Audience :: Developers",
    "TOPIC" : "Topic :: Software Development",
-   "INSTALLREQUIRES" : ["pypandoc","colorama","mysqlclient","GenPackageDoc","PythonExtensionsCollection","lxml","MySQL-python"],
```

Comment:
Hi  Ngoan,

Thank you for you  pull-request, but now I'm really confused?
Where do we get the mysql binding from?
Thank you,
 Thomas

Reply:
Hi Thomas,

**mysqlclient** is specified as `install_requires` of this package.
So, it will be installed along with this package's installation using `pip install` (on user machine).

**mysqlclient** and packages in `install_requires` configuration is required for lib/tool usage, not for building package.

Please let me know if it is still not clear for you.

Thank you,
Ngoan